dev/service-registry/client-javadsl/src/main/java/com/lightbend/lagom/internal/javadsl/registry/RegisteredService.java

```diff
@@ -26,9 +22,7 @@
  * {@code RegisteredService.of()}.
  */
 @SuppressWarnings("all")
-@ParametersAreNonnullByDefault
 @Generated({"Immutables.generator", "AbstractRegisteredService"})
```

Comment:
When I looked at findbugs 6 years ago, it wasn't under very active development.  I suspect that's still the case, so no ones really had any interest in solving the problem.